src/components/date-picker/date-picker.tsx

```diff
@@ -185,6 +185,7 @@ export class DatePicker {
     public render() {
         const inputProps: any = {
             onAction: this.clearValue,
+            trailingIcon: null,
```

Comment:
Hmm… I can't reproduce the bug on `next`… 🤔

I wonder if it has been fixed some other way since this PR was opened?

Reply:
Huh!? Seems like we haven't had a single release of Lime Elements for 16 days 😮 😄